extensions/notebook/src/intellisense/completionItemProvider.ts

```diff
@@ -197,4 +194,5 @@ export interface INewIntellisenseInfo {
 	editorUri: string;
 	cell: nb.NotebookCell;
 	notebook: nb.NotebookDocument;
+	kernel?: nb.IKernel;
```

Comment:
Do we need this to be nullable ?

Reply:
Yes, because I define the overall info object without it and set it later